src/components/safe-apps/AppFrame/index.tsx

```diff
@@ -152,15 +160,17 @@ const AppFrame = ({ appUrl, allowedFeaturesList, safeAppFromManifest }: AppFrame
           />
         </div>
 
-        <TransactionQueueBar
-          expanded={queueBarExpanded}
-          visible={queueBarVisible && !queueBarDismissed}
-          setExpanded={setExpanded}
-          onDismiss={dismissQueueBar}
-          transactions={transactions}
-        />
+        {!isNativeEmbed && (
```

Comment:
It's a bit unclear why a native Embed wouldn't have the transactionQueueBar. I remember talking to @schmanu and I think he likes the bar and even wanted to have it for the swap feature. 

Maybe a more explicit name such as "hideTransactionBar" is more appropriate? But then I see above that we are also removing the page title... and also not emitting tracking events.

Reply:
I think native swaps and native staking are both basically full screen Safe apps. So I do not see a reason to hide the queue bar if there are transactions in the queue.
Especially if some actions require multiple interactions, forcing users to switch context means that likely the widgets will load back into their original state so some context could be lost.

Reply:
I'm not against keeping it. It's just not consistent with other pages. Why donn't we show it on all pages except for the queue itself?

Reply:
Anyway, I removed the check that hides the queue bar.